opal/models.py

```diff
@@ -754,6 +754,12 @@ def category(self):
         )[0]
         return category(self)
 
+    @property
+    def stage(self):
+        s = self.stage_set.filter(stopped=None).last()
```

Comment:
I feel like I want this to be `.get()` and error if we can't set it as a constraint on the model...

Reply:
so we use last here because stage it not necessary set so I think this is ok.